tests/python_language.py

```diff
@@ -39,6 +39,10 @@ def testInheritance(self):
         self.assertEqual(alice.run, tcp.run, "Did not populate alice.run correctly")
         self.assertNotEqual(alice.logger, None, "Did not populate alice.logger correctly")
 
+    def testString(self):
```

Comment:
It will be run automatically in the test harness* - any function that begins with `test` will.

\* As long as the test file is listed in [test.py](https://github.com/mozilla-services/updatebot/blob/master/test.py#L11)